### PR TITLE
Eliminate compiler & lint warnings

### PR DESCRIPTION
**Pull request by frank-weinberg**
See the commit comments for details. I kept it to one type of warning per commit for easier review.

### CODE REVIEW

File: tests/com/carolinarollergirls/scoreboard/ExampleTest.java [hunk 1]
```diff
@@ -10,6 +11,7 @@
 public class ExampleTest {
 
 	@Test
+	@SuppressWarnings("all")
```
Comment (brian-brazil) [inline tests/com/carolinarollergirls/scoreboard/ExampleTest.java:14]: Let's just delete this test.

**Review by brian-brazil — COMMENTED** [review]
Do you want this to go in before or after your other change?

**Comment by frank-weinberg** [comment]
I don't have a preference for merge order. I expect having to do a manual merge either way.

**Comment by brian-brazil** [comment]
It's probably safest to do this one 2nd, as it's largely mechanical.

**Comment by frank-weinberg** [comment]
Given that you requested more work put into the other PR, I'd rather see this one merged before more conflicting PRs are added.

**Comment by brian-brazil** [comment]
Sure.

**Pull request merged**